sp041_20191205T1757.cdl

```diff
@@ -332,4 +332,46 @@ data:
  PLATFORM_MODEL = "Spray" ;
 
  WMO_IDENTIFIER = "4801948" ;
+
+group: CTD {
+
+  // group attributes:
+  		:long_name = "CTD sensor" ;
+  		:type = "CTD" ;
+  		:type_vocabulary = "http://vocab.nerc.ac.uk/collection/OGXXX/" ;
+  		:maker = "Seabird" ;
+  		:maker_vocabulary = "http://vocab.nerc.ac.uk/collection/OGXXX/" ;
+  		:model = "SBE 41CP" ;
+  		:model_vocabulary = "http://vocab.nerc.ac.uk/collection/OGXXX/" ;
+  		:serial_number = "32" ;
```

Comment:
Could add in the UUID to the sensor instance - otherwise it's a bit of a free text field 
An example would be -  TOOL1188_0292
The first part is the model followed by the serial number 
Can look to get a link if interested.

Reply:
Good question. I don't have an opinion here. I used here the serial number designated by Seabird itself.

Reply:
I would stick to free text here. 
I feel like it is complexifying this field to add the UUID.

Reply:
I would question if the UUID is complex as we are already requiring the model concept (TOOL1188) and this is the model plus the serial number. 

If we want an initial release, we could still do both - free text serial and keep the UUID field for the machine.

Reply:
I agree with @emmerbodc that it is not complex to define it.

I think we are not adding much value here if we are just adding two fields (concept + serial number), but if anyone has a strong opinion on having `UUID`, I have nothing against it. But please, let me know what you think so we can move on.

My vote is to leave UUID out.

Reply:
@emmerbodc and @vturpin , I think this one is also resolved, right? Otherwise, just let me know.